packages/shared/components/popups/ManageAccountPopup.svelte

```diff
@@ -32,14 +28,7 @@
             }
 
             isBusy = true
-
-            if ($isActiveLedgerProfile) {
-                promptUserToConnectLedger(_save, _cancel)
-            } else if ($isSoftwareProfile && $isStrongholdLocked) {
-                openPopup({ type: 'password', props: { onSuccess: _save } })
-            } else {
-                void _save()
-            }
```

Comment:
don't we need the `checkActiveProfileAuth` here as well?

Reply:
We are not doing anything that requires new address generation or signing.